worker/Dockerfile

```diff
@@ -1,52 +1,55 @@
-FROM climateimpactlab/miniconda3-gfortran-netcdf:v1.0.2
+## using same container as jupyter/base-notebook:python-3.7.3
+FROM ubuntu:bionic-20190612@sha256:9b1702dcfe32c873a770a32cfd306dd7fc1c4fd134adfb783db68defc8894b3c
+ARG DEBIAN_FRONTEND=noninteractive
```

Comment:
What about everything else in continuumio/miniconda3? Did none of it matter except the miniconda install?

Reply:
there's really nothing there besides miniconda: https://hub.docker.com/r/continuumio/miniconda3/dockerfile